apis/bigqueryanalyticshub/v1alpha1/bigqueryanalyticshubdataexchange_types.go

```diff
@@ -100,6 +96,10 @@ type BigQueryAnalyticsHubDataExchangeStatus struct {
 // BigQueryAnalyticsHubDataExchangeSpec defines the desired state of BigQueryAnalyticsHubDataExchange
 // +kcc:proto=google.cloud.bigquery.analyticshub.v1.DataExchange
 type BigQueryAnalyticsHubDataExchangeObservedState struct {
+	// Output only. The resource name of the data exchange.
+	//  e.g. `projects/myproject/locations/US/dataExchanges/123`.
+	Name *string `json:"name,omitempty"`
```

Comment:
You _could_ also just drop it, as long as we are populating status.externalRef

Reply:
😱 actually I like that idea; today this is outputted in the format of an externalRef so it's actually redundant in this case.